pkg/component/operator/audio/v0/task_detect_activity.go

```diff
@@ -0,0 +1,125 @@
+package audio
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/go-audio/audio"
+	"github.com/go-audio/wav"
+	"github.com/mjibson/go-dsp/fft"
+	"github.com/streamer45/silero-vad-go/speech"
```

Comment:
This also depends on `ONNX` so building the code on the local dev env fails, which might create confusion. Bulding the code as I write it is a usual step to detect errors early.

```
go build ./cmd/main
# github.com/streamer45/silero-vad-go/speech
In file included from ../../../../pkg/mod/github.com/streamer45/silero-vad-go@v0.2.1/speech/detector.go:5:
./ort_bridge.h:1:10: fatal error: 'onnxruntime_c_api.h' file not found
    1 | #include "onnxruntime_c_api.h"
      |          ^~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
Right. The problematic `include` is all the way from the dependent package. Will add the build tag correspondingly.